readme.md

```diff
@@ -181,11 +181,11 @@
 - [React](https://github.com/enaqx/awesome-react) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay) - Framework for building data-driven React apps.
 - [Web Components](https://github.com/mateusortiz/webcomponents-the-right-way)
-- [Polymer](https://github.com/Granze/awesome-polymer)
+- [Polymer](https://github.com/Granze/awesome-polymer) - JavaScript library for progressive web apps.
```

Comment:
Polymer does not make for progressive web apps, rather it is for making WebComponents (as per the upcoming HTML5 spec for WebComponents).

Reply:
Is "JavaScript library for WebComponents" better?

Reply:
> Is "JavaScript library for WebComponents" better?

Sounds good.  :-)
Maybe "JavaScript library to develop WebComponents" though?

But overall Polymer actually makes Progressive Web apps more difficult to do, it is definitely not something that allows for easy fallbacks.  ^.^;